github/client.go

```diff
@@ -143,6 +143,16 @@ func (c *Client) ReadReleaseByTag(ctx context.Context, tkn string, repoURL url.U
 	return model.ToSvcGithubRelease(rls)
 }
 
+// GenerateGithubReleaseURL generates a URL to the release page on GitHub for a given repository and tag name
+func (c *Client) GenerateGithubReleaseURL(repoURL url.URL, tagName string) (string, error) {
```

Comment:
If https://github.com/jan-zabloudil/release-manager/pull/114 is merged, I will update it to accept `svcmodel.GithubRepo`